src/app/pages/dashboard/components/practice-test-report-card/__tests__/practice-test-report-card.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { async, TestBed } from '@angular/core/testing';
+import {
+  AlertController, ModalController,
+} from '@ionic/angular';
+import { AlertControllerMock, ModalControllerMock } from 'ionic-mocks';
+import { StoreModule } from '@ngrx/store';
+import { configureTestSuite } from 'ng-bullet';
+// import { PracticeTestReportCardComponent } from '../practice-test-report-card';
+// import { journalReducer } from '../../../../../modules/journal/journal.reducer';
+
+describe('PracticeTestReportCard ', () => {
+
+  configureTestSuite(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        StoreModule.forRoot({}),
+      ],
+      providers: [
+        { provide: AlertController, useFactory: () => AlertControllerMock.instance() },
+        { provide: ModalController, useFactory: () => ModalControllerMock.instance() },
+      ],
+    });
+  });
+
+  beforeEach(async(() => {
+  }));
+
+  xdescribe('Class', () => {
```

Comment:
is this test failing?

Reply:
No, I put that there before I got rid of the contents that used NavController, which we no longer use